pkg/email/templates/password_changed.html.tmpl

```diff
@@ -0,0 +1,19 @@
+{{- define "title" -}}
+Password Changed
+{{- end -}}
+
+{{- define "preview" -}}
+The password of your user "{{ .Notification.EntityIds.IDString }}" has just been changed.
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+  Dear {{ .ReceiverName }},
+</p>
+<p>
+The password of your user "{{ .Notification.EntityIds.IDString }}" on {{ .Network.Name }} has just been changed.
+</p>
+<p>
+If this was not done by you, please contact your administrators as soon as possible.
```

Comment:
```suggestion
If this was not requested by you, please contact your administrators as soon as possible.
```

Reply:
It's not a pending request, it was done.